### PR TITLE
Fires Bluespace Artillery at the captain if he tries to put the disk in the safe.

### PR DESCRIPTION
**Pull request by ShizCalev**
Closes #34327
Closes #34332

### CODE REVIEW

File: code/game/objects/structures/safe.dm [hunk 1]
```diff
@@ -157,6 +157,17 @@ FLOOR SAFES
 
 /obj/structure/safe/attackby(obj/item/I, mob/user, params)
 	if(open)
+		if(istype(I, /obj/item/disk/nuclear) && iscarbon(user))
+			var/mob/living/carbon/C = user
+			if(C.mind && C.mind.assigned_role == "Captain")
```
Comment (coiax) [inline code/game/objects/structures/safe.dm]: I mean, I know this is a meme PR, but didn't @AnturK say that checking this sort of thing was bad practice? Or is that only with antagonists?

**Review by Cyberboss — APPROVED** [review]

File: code/game/objects/structures/safe.dm (hunk 1 above)
Comment (ma44) [reply code/game/objects/structures/safe.dm]: This can probably be used to check if say a ling is actually a captain or not.

**Comment by Xhuis** [comment]
This is the best solution

**Comment by ma44** [comment]
But why would the captain get BSAed but not the rest of the crew? 🤔 
It's not like there's a piece of paper that says 'don't put the damm disk in the safe'

**Comment by ShizCalev** [comment]
Captains being the ultimate authority on the station, appointed by CentCom itself to the command of a Plasma Research Station, should be held to higher standards than other cucks. :^)

**Comment by optimumtact** [comment]
stop, we're adding a fuckign way to get into the safe christ

**Comment by ShizCalev** [comment]
CENSORSHIP

**Pull request closed without merging**